mappings/net/minecraft/block/Material.mapping

```diff
@@ -2,55 +2,98 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15911 solid Z
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
+		COMMENT Material for cobweb-like blocks.
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Hard solid nonmetallic mineral matter of which rock is made, especially as a building material.
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+		COMMENT Material for water bubble streams.
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
+		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
+		COMMENT Material for sponges.
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
+		COMMENT Material for redstone lamps.
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
+		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
+		COMMENT Material for the water block.
```

Comment:
Is this really necessary? It's not enough that the name of the field is `WATER`, we have to specify that it's for `water` too?

Reply:
Likewise applies to other jds in this PR.

Reply:
Did it for consistency, and because blocks could be added to this material at any time, so would need the JD/rename at that point.

Reply:
Doesn't address the issue. These comments are all redundant.